creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -0,0 +1,54 @@
+const promClient = require('prom-client')
+
+const METRIC_PREFIX = 'cn_'
+
+/**
+ * @notice Counter and Summary metric types are currently disabled, since we should almost never be using them.
+ *  - Gauge should always be used over Counter since it can be decreased, with no performance overhead.
+ *  - Histogram should always be used over Summary, per docs below
+ * https://prometheus.io/docs/tutorials/understanding_metric_types/
+ */
+const MetricTypes = Object.freeze({
+  GAUGE: promClient.Gauge,
+  HISTOGRAM: promClient.Histogram
+  // COUNTER: promClient.Counter,
+  // SUMMARY: promClient.Summary
+})
+
+const MetricNames = Object.freeze({
+  STORAGE_PATH_SIZE_GAUGE: 'storage_path_size_bytes',
+  STORAGE_PATH_USED_GAUGE: 'storage_path_used_bytes',
+  ROUTE_POST_TRACKS_DURATION_SECONDS_HISTOGRAM:
+    'route_post_tracks_duration_seconds'
+})
```

Comment:
I'm +0 on using _GAUGE and _HISTOGRAM suffixes. (Also, it's the end of the week so it could just be me.)

I like how that enables us to tell if we should use `.set()` or `.observe()`. Any other reason you chose this pattern?

Reply:
yep, its just a usability thing so caller can easily identify the metric type
also noting that this suffix is only on the var name, not the string value

Reply:
Indeed, caught that it was only on the var name and I think that's probably where my hesitancy arose from.

With fresh eyes, this looks good!